test/conformance/testing/include/uur/fixtures.h

```diff
@@ -22,14 +22,14 @@
     }                                                                          \
     (void)0
 
-#define UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(ret)                                 \
+#define UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(ret) do {                            \
     auto status = ret;                                                         \
     if (status == UR_RESULT_ERROR_UNSUPPORTED_FEATURE ||                       \
         status == UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION) {                   \
         GTEST_SKIP();                                                          \
     } else {                                                                   \
         ASSERT_EQ(status, UR_RESULT_SUCCESS);                                  \
-    }
+    } } while(0)                                                               \
```

Comment:
I know it's semantically the same but I'd prefer just a scope like https://github.com/oneapi-src/unified-runtime/pull/2193/files#diff-3c41a24f2744a3a369eebf302dd6359004331bb34e5828ea2033c4aeafc5ca3aR26

Reply:
do .. while(0) is traditionally used to ensure semicolons are handled correctly. For example:

```c++
if (something)
  UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(ret);
else
  something_else();
```

https://stackoverflow.com/questions/154136/why-use-apparently-meaningless-do-while-and-if-else-statements-in-macros

Reply:
ah I see, it's ugly but if it works it works